aws/components/lb/setup.ftl

```diff
@@ -675,9 +678,32 @@
     [/#if]
 
     [#switch engine ]
-        [#case "network"]
         [#case "application"]
+            [#if deploymentSubsetRequired(LB_COMPONENT_TYPE, true) ]
+                [#-- Create a WAF ACL if required --]
+                [#if wafAclResources?has_content ]
+                    [@createWAFAclFromSecurityProfile
+                        id=wafAclResources.acl.Id
+                        name=wafAclResources.acl.Name
+                        metric=wafAclResources.acl.Name
+                        wafSolution=wafSolution
+                        securityProfile=securityProfile
+                        occurrence=occurrence
+                        regional=true
+                    /]
 
+                    [#if !cfResources?has_content]
```

Comment:
Think this might be a copy/paste error? Don't think we have support for cloudfront in the lb component?